Casks/workbrew.rb

```diff
@@ -10,12 +10,22 @@
   pkg "Workbrew-#{version}.pkg"
 
   preflight do
-    next if ENV["USER"] == "workbrew"
+    if ENV["USER"] != "workbrew"
+      raise <<~EOS
+        The Workbrew Installer must be run manually the first time.
+        This Cask is only used for upgrades. Download it and follow the instructions:
+          #{Formatter.url("https://console.workbrew.com")}
+      EOS
+    end
+
+    workbrew_directory = Pathname.new("/opt/workbrew")
+    workbrew_api_key_directory = workbrew_directory/"home/Library/Application Support/com.workbrew.workbrew-agent"
+    return if (workbrew_api_key_directory/"device_api_key").exist? || (api_key_directory/"api_key").exist?
```

Comment:
```suggestion
    return if (workbrew_api_key_directory/"device_api_key").exist? || (workbrew_api_key_directory/"api_key").exist?
```
or, more readable to me:
```suggestion
    return if %w[device_api_key api_key].any? { |file| (workbrew_api_key_directory/file).exist? }
```

Reply:
@carlocab Moving to https://github.com/Workbrew/homebrew-tap/pull/24